### PR TITLE
MAINT-28323: Fix display of long application title.

### PR DESCRIPTION
**Pull request by mdiha**
I mad sure to set the application title div width according to the different view port sizes.

### CODE REVIEW

File: app-center-webapps/src/main/webapp/vue-apps/adminSetup/components/AdminSetupList.vue [hunk 1]
```diff
@@ -64,7 +64,7 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
               v-exo-tooltip.bottom.body="props.item.title.length > 22 ? props.item.title : ''"
               class="text-md-center tableAppTitle"
             >
-              {{ props.item.displayName ? props.item.displayName : props.item.title}}
+              {{ props.item.displayName ? props.item.displayName : props.item.title }}
```
Comment (ahamdi) [inline app-center-webapps/src/main/webapp/vue-apps/adminSetup/components/AdminSetupList.vue:67]: Is this change needed ?

**Review by ahamdi — CHANGES_REQUESTED** [review]

File: app-center-webapps/src/main/webapp/vue-apps/adminSetup/components/AdminSetupList.vue (hunk 1 above)
Comment (mdiha) [reply app-center-webapps/src/main/webapp/vue-apps/adminSetup/components/AdminSetupList.vue:67]: It is a warning on npm watch so i fixed it.

**Review by ahamdi — APPROVED** [review]
Good Thanks
